src/Status/StatusCard/StatusCard.tsx

```diff
@@ -32,11 +37,21 @@ const card = ({ status, name, iconTooltipText }: StatusCardProps) => {
           iconTooltipText={iconTooltipText}
         />
         <Box
-          sx={{ display: "flex", flexDirection: "column", marginLeft: "8px" }}
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            marginLeft: "8px"
+          }}
         >
-          <Typography variant="h6" color="text.primary" fontWeight={600}>
-            {name}
-          </Typography>
+          <TruncatedTooltip multiline={1}>
+            <Typography
+              variant={titleVariant}
+              color="text.primary"
+              fontWeight={600}
```

Comment:
@Sowbhagya-ipg do you want a font-weight of 600 for every typography variant, or do you want the default font-weight for each variant?

Reply:
Had a chat about this before implementing and she was fine with the font-weight 600